drivers/Zigbee2MQTTDevice.js

```diff
@@ -326,7 +326,7 @@ module.exports = class Zigbee2MQTTDevice extends Device {
 					if (message.toString() === '') return;
 					const info = JSON.parse(message);
 					// Map the incoming value to a capability or setting
-					if (topic.includes(this.deviceTopic)) {
+					if (topic == this.deviceTopic) {
```

Comment:
you should use `===` to compare strings if you are not specifically trying to check for ref equality

Reply:
My bad, I'm used to other programming languages where === is a pointer reference check